File: 7 Kyu/Python/flatten_and_sort_an_array.py
# Given a two-dimensional array of integers, return the flattened version of the array with all the integers in the sorted (ascending) order.

def flatten_and_sort(array):
    num = []
    
    for arr in array:
        for number in arr:
            num.append(number)
    num.sort()
    return num

File: 7 Kyu/Javascript/flatten_and_sort_an_array.js
// Given a two-dimensional array of integers, return the flattened version of the array with all the integers in the sorted (ascending) order.

function flattenAndSort(array) {
    let sorted = [];
    
    for (arr of array) {
        for (num of arr) {
            sorted.push(num)   
        }
    }
    
    return sorted.sort((a, b) => a - b);
}